mongodbatlas/config.go

```diff
@@ -38,7 +39,7 @@ func (c *Config) NewClient(ctx context.Context) (interface{}, diag.Diagnostics)
 
 	client.Transport = logging.NewTransport("MongoDB Atlas", transport)
 
-	optsAtlas := []matlasClient.ClientOpt{matlasClient.SetUserAgent("terraform-provider-mongodbatlas/" + ProviderVersion)}
+	optsAtlas := []matlasClient.ClientOpt{matlasClient.SetUserAgent("terraform-provider-mongodbatlas/" + version.ProviderVersion)}
```

Comment:
[q] this is the user agent we do for the cli

https://github.com/mongodb/mongocli/blob/6ac1f831b2a7d4ae60f4a7d025e4ab3c8da50d79/internal/store/store.go#L35

should we make something similar, ie: include os and arch?

Reply:
Yeah that makes sense.  Will update later, but if you want to make the change directly on the PR it would be even better, since I won't be in front of a computer for the next few hours.  Thanks!

Reply:
On second thought @gssbzn, if we made this change, the UA would be reported as:
```
 userAgent: terraform-provider-mongodbatlas/0.9.1-29-g13eb1636 (darwin;amd64) go-mongodbatlas/0.10.1 (darwin;amd64) 
```

(since we add the OS/ARCH via the go client)

Reply:
I'm going to merge the PR as is / I don't think duplicating that info makes sense.

Reply:
totally forgot we were doing that in the client, merge away